x-pack/plugins/security_solution/public/timelines/components/side_panel/event_details/index.tsx

```diff
@@ -88,23 +98,24 @@ const EventDetailsPanelComponent: React.FC<EventDetailsPanelProps> = ({
     skip: !expandedEvent.eventId,
   });
 
-  const [isHostIsolationPanelOpen, setIsHostIsolationPanel] = useState(false);
-
+  const {
+    services: { osquery },
+  } = useKibana();
+  const [isActivePanel, setIsActivePanel] = useState<number | null>(null);
```

Comment:
`is` indicates the value to be a type of boolean, can we change it to something less confusing?